rdt/hyper_transformer.py

```diff
@@ -23,9 +24,14 @@ class Config(dict):
 
     def __repr__(self):
         """Pretty print the dictionary."""
+        transformers_repr = {}
```

Comment:
I honestly prefer the one-liner implementation, I find it more concise and faster to read. It doesn't matter much though, so you can use whichever way you prefer.

Reply:
Yes me too but the one-liner implementation was crashing for tuples as key that's why I changed it. I got this error:
`TypeError: keys must be str, int, float, bool or None, not tuple`